3-enrich/stream-enrich/core/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/model.scala

```diff
@@ -44,10 +47,16 @@ object model {
     partitionKey: String
   )
   final case class KinesisBackoffPolicyConfig(minBackoff: Long, maxBackoff: Long)
-  sealed trait SourceSinkConfig
+  sealed trait SourceSinkConfig {
```

Comment:
Agnostic config doesn't depend on a cloud platform and used for Kafka, NSQ and Stdin whereas SourceSinkConfig is used for Kinesis, hence it isn't platform agnostic. I'll fix naming and add scaladocs and see if I can further improve without touching configuration.

Reply:
> Base source config has GCP for some reasons.

that is because stream-enrich-kinesis might run an enrichment where enrichment requires a file hosted privately in GCS and stream-enrich-kinesis depends on SourceSinkConfig, not SourceSink**Agnostic**Config, since it is platform dependent and not agnostic

Repeated above but I'm just writing to answer the question for all reviewers, I'll fix naming hopefully and add scaladocs to ease understanding

Reply:
For clarity, I put an explanation last week just to answer the question, these comments are invalid after addressing Anton's feedback, since both types are removed.